### PR TITLE
[rush-pnpm] Fix rush-pnpm patch-commit command

### PR DESCRIPTION
**Pull request by L-Qun**
## Details

After upgrading the rush version(5.131.0) in tiktok monorepo, we encountered an error when running `rush-pnpm --subspace subspaceName patch-commit xxxx`

img_v3_02di_2ae44f28-61d6-4bcf-91e1-c2968dbfb3hu

After taking a close look at the rush source code, I found that the path to the folder related to the pnpm patches was incorrect. Therefore, I fixed this issue in that MR

## How it was tested

To test the implementation, the following commands were executed:

1. `rush install -t .`
2. `rush-pnpm --subspace subspaceName patch xxx`
3. `rush-pnpm --subspace subspaceName patch-commit xxx`

No errors were encountered.

## Impacted documentation

None

### CODE REVIEW

**Comment by L-Qun** [comment]
@iclanton Pete is on vacation. We need your help with the review

**Review by D4N14L — COMMENTED** [review]
@L-Qun have you validated that this still works in non-subspaces scenarios (or I guess, the default subspace)?

**Comment by L-Qun** [comment]
> @L-Qun have you validated that this still works in non-subspaces scenarios (or I guess, the default subspace)?

@D4N14L Yeah, I tested it. Everything works well

image

image

**Review by D4N14L — APPROVED** [review]

File: common/changes/@microsoft/rush/fix-pnpm-patch_2024-08-09-10-38.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix rush-pnpm patch-commit",
```
Comment (iclanton) [inline common/changes/@microsoft/rush/fix-pnpm-patch_2024-08-09-10-38.json:5]:
```suggestion
      "comment": "Fix an issue where `rush-pnpm patch-commit` would not correctly resolve patch files when the subspaces feature is enabled.",
```

**Review by iclanton — APPROVED** [review]

**Comment by L-Qun** [comment]
@iclanton Could you help to publish a new rush version

**Comment by iclanton** [comment]
@L-Qun Published in Rush v5.131.3

**Comment by L-Qun** [comment]
> @L-Qun Published in Rush v5.131.3

Thanks @iclanton

**Comment by iclanton** [comment]
@L-Qun - Looks like this PR introduced a breaking change. In the non-subspaces scenario, the patches folder is now `common/config/rush/pnpm-patches`, and before it was `common/config/pnpm-patches`

**Pull request merged**